test/bowlingGame.test.js

```diff
@@ -0,0 +1,105 @@
+const BowlingGame = require("../bowlingGame");
+
+describe("BowlingGame", () => {
+  const game = new BowlingGame();
+
+  it("can be initialised with no argument", () => {
+    expect(game.rollList).toEqual([]);
+  });
+
+  it("can store the rolls as an array", () => {
+    game.roll(8);
+    game.roll(1);
+    game.roll(3);
+    expect(game.rollList).toEqual([8, 1, 3]);
+  });
```

Comment:
Don't test the state of your class.
You should be testing behaviour of a class, which means you only should test a class through its methods that either print or return based on the state of the class.

The correct way here is to make some rolls, then check the score.
The reason for this is that the behaviour is stable - meaning the score would always be the same given the same rolls - however, you could change the class implementation to remove the rollList, or store the rolls differently like in a hash. By having tests that look at the implementation rather than the behaviour, you making it a lot harder to refactor you code.

Reply:
where refactor = change implementation without changing behaviour.

Reply:
It can be useful to think of behaviour as input/output